### PR TITLE
Clean up excitatory/inhibitory input port semantics

### PR DESCRIPTION
**Pull request by clinssen**
Clean up the interpretation of signs in spike events. This ties in with the ``excitatory`` and ``inhibitory`` input port qualifiers in the NESTML model.

Please see the ``neurons_in_nestml.rst`` file for documentation about the change.

### CODE REVIEW

File: doc/nestml_language/neurons_in_nestml.rst [hunk 1]
```diff
@@ -19,23 +19,57 @@ Neuronal interactions
...
+
+   input:
+     # [...]
+     AMPA_spikes pA <- excitatory spike
+     GABA_spikes pA <- inhibitory spike
+   end
+
+In this case, spike events that have a negative weight are routed to the ``GABA_spikes`` input port, and those that have a positive weight to the ``AMPA_spikes`` port.
+
+It is equivalent if either both ``inhibitory`` and ``excitatory`` are given or neither of them. If only a single one is given, another input port has to be present and specify the inverse keyword.
```
Comment (pnbabu) [inline doc/nestml_language/neurons_in_nestml.rst:58]: Does this mean a model cannot have an input port with only either of the qualifiers?

**Review by pnbabu — COMMENTED** [review]
There is a merge conflict now. Could you resolve it?

File: doc/nestml_language/neurons_in_nestml.rst (hunk 1 above)
Comment (clinssen) [reply doc/nestml_language/neurons_in_nestml.rst:58]: It's actually legal to have any combination (both, neither, or just one qualifier). I've updated this in the documentation now. Cheers!

**Review by pnbabu — APPROVED** [review]
Thanks! Looks good to me.

**Pull request merged**